internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestAdminOperationVersionDetection.java

```diff
@@ -82,7 +83,7 @@ public void testGetAdminOperationVersionForParentControllers() {
     assertEquals(response.getCluster(), clusterName);
   }
 
-  @Test(timeOut = TEST_TIMEOUT)
+  @Test(timeOut = TEST_TIMEOUT * 2)
```

Comment:
BTW, are these changes needed?

Reply:
Yes, I need more time to have the retries for standby finder.